docs/extras/beta/index.md

```diff
@@ -0,0 +1,79 @@
+
+We are very excited about OpenTrials, and its potential role as part of the ecosystem to change the way clinical trials are conducted and reported on.
+
+In this post we'll look at the features that are available in the OpenTrials beta release, and what is in the pipeline on the road to a proper v1 release.
+
+## Features
+
+OpenTrials is designed as a modular platform, employing a set of loosely coupled components that handle distinct tasks from the collection of data from external sources, through to the exposing of data for consumption via the API or a UI.
+
+In the [announcement post](http://opentrials.net/2016/02/10/opentrials-technical-roadmap/) for the platform, we presented two diagrams that describe the architecture of the platform, and the data model that OpenTrials employs. The design presented then has not significantly changed, and you can see those diagrams below for convenience.
+
+<img src="images/architecture.png" width="600" style="text-align: center; display: block; margin: 0 auto;" title="OpenTrials Architecture Overview">
+
+<img src="images/model.png" width="600" style="text-align: center; display: block; margin: 0 auto;" title="OpenTrials Data Model Overview">
+
+What those diagrams do not reveal is the actual feature set exposed to users, so we'll highlight the major features here.
+
+### API
+
+The API is currently at **v1**. The base endpoint for the API is [here](http://api.opentrials.net/v1/), and interactive documentation for the API can be found [here](http://api.opentrials.net/v1/docs/).
+
+The API exposes several RESTful endpoints that allow querying the primary entities of the OpenTrials data model. This is particularly useful for creating applications that paginate through related data, or where the application has identifiers for particular objects and wants to query all information for those objects.
+
+The API also exposes a search endpoint, which is backed by Elasticsearch. This allows for deep queries into the entire database, and does not necessarily require any knowledge of the data model, or the relations between entities, to yield useful results. We've found that, given the nature of the data itself, the search endpoint is the most useful endpoint for regular use.
+
+Our API implements the [Open APIs specification](https://openapis.org). One of the many great benefits of implementing against Open APIs is that client libraries can be auto-generated from the spec itself. So, fire up a REPL in [Python](https://github.com/Yelp/bravado), [JavaScript](https://github.com/swagger-api/swagger-js) or any other language that has a Swagger/Open APIs implementation, and start playing with the data.
+
+### Explorer UI
+
+The [Explorer UI](https://explorer.opentrials.net) is the main portal into the OpenTrials data. It is based on a search-driven user experience, and once a user reaches a trial page, opens up further navigation into the related entities for that trial.
```

Comment:
Unfortunately, we don't have HTTPS enabled yet, so this should be `http://explorer.opentrials.net`